Nomenclature (#1170)
OctopusDeploy/docs: docs/getting-started/index.md

The guide below provides more information on how to [package your software](#package-software).

## Terminology

> As start using Octopus Deploy, you are going to learn a lot of new terms.  So much so it can feel overwhelming.  You can view a list of common terms on the [terminology page](docs/getting-started/terminology.md).

Reviewer: Slight tweak, also I removed overwhelm so as not to pre-overwhelm readers :)
```suggestion
As you start using Octopus Deploy, you will learn a lot of new terms. You can view a list of common terms on the [terminology page](docs/getting-started/terminology.md).
```
